modules/components/tag-input/tag-input.ts

```diff
@@ -311,6 +311,11 @@ export class TagInputComponent extends TagInputAccessor implements OnInit, After
      */
     @ViewChild(TagInputForm) public inputForm: TagInputForm;
 
+  /**
+   * @name targetElement
+   */
+    @ViewChild('tagsContainer') public targetElement: ElementRef;
```

Comment:
Rename the variable to `tagsContainer` as well. `targetElement` is a very generic term that doesn't help understanding what the variable is.